src/main/scala/org/example/sinks/ObjectMapperWrapper.scala

```diff
@@ -0,0 +1,18 @@
+package org.example.sinks
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+
+object ObjectMapperWrapper extends java.io.Serializable {
+  // this lazy is the important trick here
+  // @transient adds some safety in current Scala (see also Update section)
+  // https://stackoverflow.com/a/48385665
+  @transient lazy val mapper = {
+    val mapper = new ObjectMapper
+    mapper.registerModule(DefaultScalaModule)
```

Comment:
`DefaultScalaModule` is not available in `org.apache.flink.shaded.jackson2.com.fasterxml.jackson...`. Maybe it is better to be consistent here and use non-shaded `com.fasterxml.jackson...`